cfgov/data_research/templates/data_research/conference_notify_subject.txt

```diff
@@ -1 +1 @@
-[{{ conference_name }}] Attendance {{ at_capacity|yesno:"Full,Update" }}
+[{{ conference_name }}] Attendance Update{{ at_capacity|yesno:" (in-person capacity reached)," }}
```

Comment:
Is there an extra trailing comma after `reached)`?

Reply:
No, it is the separation between the `yes` output and the `no` output (which should be none). Very strange Django filter, this is.

Reply:
Ah, right. Thanks for the correction.